Sources/FuturedKit/Publisher+Result.swift

```diff
@@ -0,0 +1,16 @@
+import Combine
+
+extension Publisher {
+    /// Transforms all elements and failures to ``Result``.
```

Comment:
```suggestion
    /// Transforms all elements and failures to `Result`.
```

Double quotes are for documentation links, but the don't work for type which are defined outside of the package.